app-admin/elektra/elektra-0.8.15.ebuild

```diff
@@ -97,10 +100,10 @@ multilib_src_configure() {
 	fi
 
 	mycmakeargs=(
+		"-DBUILD_PDF=OFF"
 		"-DBUILD_SHARED=ON"
 		"-DPLUGINS=${my_plugins}"
 		"-DTOOLS=${my_tools}"
-		"-DLATEX_COMPILER=OFF"
 		"-DTARGET_CMAKE_FOLDER=share/cmake/Modules"
 		$(multilib_is_native_abi && cmake-utils_use doc BUILD_DOCUMENTATION \
 			|| echo -DBUILD_DOCUMENTATION=OFF)
```

Comment:
can we simply this please? `-DBUILD_DOCUMENTATION=$(multilib_is_native_abi && usex doc || echo no)`